samples/boards/mec15xxevb_assy6853/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
The path for this application is wrong.
```
Create a path like this samples/boards/mec15xxxevb_assy6853/power_management
```
Place the power management application there, and in the future we can have other apps there too

Reply:
Wouldn't call it wrong, but I see the potential. Changed